support-frontend/assets/pages/aus-moment-map/components/blurb.tsx

```diff
@@ -8,11 +7,13 @@ const enableSupporterCount = true;
 const useSupportersCount = () => {
 	const [supportersCount, setSupportersCount] = React.useState(0);
 	const supportersCountEndpoint = '/supporters-ticker.json';
+
 	React.useEffect(() => {
+		// eslint-disable-next-line @typescript-eslint/no-unnecessary-condition -- if check here is supposedly unnecessary
```

Comment:
so the issue is the 

```ts
enableSupporterCount = true;
```

which typescript then uses to assign the literal type `true` to `enableSupportersCount`. We can force typescript to just class it as a `boolean` with something like:

```ts
enableSupportersCount = true as boolean;
```